static/javascript/launch.js

```diff
@@ -367,7 +366,7 @@ function getInputField(
 }
 
 async function loadSDSDatasetMetadata(survey_id, period_id) {
```

Comment:
We could possibly pass in `sdsEnabled`(as sds_enabled) so we don't rely on correct scopes and  it would make it less error prone. Also we would then avoid using camel case and snake case on the same line so more consistency across the codebase.